#!/usr/bin/env python
PACKAGE = "racecar_behaviors"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("color_hue",   int_t,    0, "Hue value", 107,  0, 180)
gen.add("color_range", int_t,    0, "Hue range (+- Hue value)", 10,  0, 50)
gen.add("color_saturation", int_t,    0, "Minimum saturation", 50,  0, 255)
gen.add("color_value", int_t,    0, "Minimum value", 0,  0, 255)
gen.add("border",      int_t,    0, "Border: ignore detections to close to border of image.", 15,  0, 100)

exit(gen.generate(PACKAGE, "racecar_behaviors", "BlobDetector"))
